start.sh

```diff
@@ -1,2 +1,3 @@
 #!/bin/sh
-ruby controllers/users.ru
+#ruby controllers/users.ru
```

Comment:
Why was this commented out with the '#'?

Reply:
Temporarily as this was just a placeholder, eventually we'll flesh out the users controller.